index.html

```diff
@@ -46,9 +46,9 @@ <h1>
           you see and have a project you need coded, don’t hestiate to contact
```

Comment:
**Kindly change the title of your project:**
- There is a page title, `<title>` in the `<head>` of the page.
It adequately and briefly describes the content of the page. For example:
Poor titles:
- My Portfolio

**_Instead use:_**
- Rachidelaid Portfolio or Rachidelaid full-stack webdeveloper